08_files_lab/01.py

```diff
@@ -1,6 +1,17 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+'''
+Usage: 01.py <targetFile> <sourceFile>
+Appends the content of <sourceFile> to the end of <targetFile>
+'''
+
+import sys
+import os
+
+if len(sys.argv) != 3 or not os.path.exists(sys.argv[1]) or not os.path.exists(sys.argv[2]):
+    print 'Usage: %s <targetFile> <sourceFile>' % sys.argv[0]

```

Comment:
Can also write the error message as an argument to sys.exit for the same effect:

```
sys.exit('Usage: %s <targetFile> <sourceFile>' % sys.argv[0])`
```